events.go

```diff
@@ -79,6 +107,11 @@ func (e *EntryNotified) EventName() string {
 	return eventEntryNotified
 }
 
+func (e *EntryNotified) String() string {
```

Comment:
Can you add a test for this, please?

Reply:
Where can I write test about that? There is `events_tests.go` file but I couldn't be sure.

Reply:
`events_tests.go` makes sense 👍